stable/artifactory/templates/artifactory-networkpolicy.yaml

```diff
@@ -0,0 +1,34 @@
+{{- range .Values.networkpolicy }}
+apiVersion: extensions/v1beta1
+kind: NetworkPolicy
+metadata:
+  name: {{ template "artifactory-ha.fullname" $ }}-{{ .name }}-networkpolicy
```

Comment:
Remove all `-ha`. This is failing the CI.

Reply:
done